diktat-ruleset/src/main/kotlin/org/cqfn/diktat/util/FileUtils.kt

```diff
@@ -0,0 +1,36 @@
+/**
+ * This class contains util methods to operate with java.nio.file.Path
+ */
+
+package org.cqfn.diktat.util
+
+import java.nio.file.InvalidPathException
+import java.nio.file.Path
+import java.nio.file.Paths
+import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.PathWalkOption
+import kotlin.io.path.exists
+import kotlin.io.path.walk
+
+/**
+ * Create a matcher and return a filter that uses it.
+ *
+ * @param glob glob pattern to filter files
+ * @return a sequence of files which matches to [glob]
+ */
+@OptIn(ExperimentalPathApi::class)
+fun Path.walkByGlob(glob: String): Sequence<Path> =
+    fileSystem.getPathMatcher("glob:$glob")
+        .let { matcher ->
+            this.walk(PathWalkOption.INCLUDE_DIRECTORIES)
+                .filter { matcher.matches(it) }
+        }
+
+/**
+ * @return path or null if path is invalid or doesn't exist
+ */
+fun String.tryToPath(): Path? = try {
```

Comment:
Up to you, but, IMO, 

- either the name should reflect that the path is returned only if it exists (i.e. `tryToPathIfExists`),
- or the function should also accept a `Path` predicate (`(Path) -> Boolean`) and be invoked as
```kotlin
"C:\\autoexec.bat".tryToPathIf(Path::exists)
```

Reply:
renamed